app/lib/additional_dependents_pdf.rb

```diff
@@ -0,0 +1,37 @@
+class AdditionalDependentsPdf
+  include PdfHelper
+
+  def source_pdf_name
+    "tax_documents/additional_dependents"
+  end
+
+  def initialize(submission)
+    # For some PDF fields, use values from the database b/c the XML values are truncated or missing.
+    @xml_document = SubmissionBuilder::Ty2021::Documents::Irs1040.new(submission).document
+  end
+
+  def hash_for_pdf
+    answers = {}
+    dependent_nodes = @xml_document.search("DependentDetail")
+    answers.merge!(dependents_info(dependent_nodes[4..])) if dependent_nodes.length > 4
+    answers
+  end
+
+  private
+
+  def dependents_info(dependent_nodes)
+    answers = {}
+    dependent_nodes.each_with_index do |dependent, index|
+      answers["DependentNameRow#{index + 1}"] = [dependent.at("DependentFirstNm").text, dependent.at("DependentLastNm").text].join(" ")
+      answers["TINRow#{index + 1}"] = dependent.at("DependentSSN").text
+      answers["RelationshipRow#{index + 1}"] = dependent.at("DependentRelationshipCd").text
+      answers["CTCRow#{index + 1}"] = xml_check_to_bool(dependent.at("EligibleForChildTaxCreditInd")) ? "Yes" : nil
+      answers["OTCRow#{index + 1}"] = xml_check_to_bool(dependent.at("EligibleForODCInd")) ? "Yes" : nil
```

Comment:
This calls it ODC (which makes sense to me, other dependents credit aka credit for other dependents).

The PDF calls it "OTC" but I think it should be "ODC".

<img width="1354" alt="image" src="https://user-images.githubusercontent.com/67708639/168919458-85708eef-9aad-4b6f-af83-d4006b328f3d.png">

Reply:
Huh, interesting -- the PDF calls it `OTC` in the form field but the XML calls it `ODC`.

Reply:
good catch!!